torch/_inductor/codegen/cpp_flex_attention_template.py

```diff
@@ -1110,3 +1111,38 @@ def codegen_allocate_buffer(self, buffer_name: str, buffer_dtype, buffer_size):
                 buffer_size=buffer_size,
             )
         )
+
+    def micro_gemm_define(self, kernel_name: str):
+        from torch._inductor.codegen.cpp_gemm_template import (
+            CppTemplateKernel,
+            parallel_num_threads,
+        )
+        from torch._inductor.codegen.cpp_micro_gemm import CppMicroGemmFP32Vec
+        from torch._inductor.virtualized import V
+
+        GemmBlocking = namedtuple("GemmBlocking", ["block_m", "block_n", "block_k"])
```

Comment:
I think we have defined `GemmBlocking` in MicroGEMM, can we reuse it?

Reply:
Reused.